tests/fixtures/Search.2.0.ethiopic.json

```diff
@@ -650,142 +650,6 @@
                 "downloads": "0",
                 "final_weight": "30.0"
             }
-        },
-        {
-            "id": "tir_et_uni_ez",
-            "name": "Tigrigna.ET EZ",
-            "jsFilename": "tir_et_uni_ez.js",
-            "platformSupport": {
-                "desktopWeb": "full",
-                "ios": "basic",
-                "android": "basic"
-            },
-            "version": "1.0",
-            "license": "mit",
-            "languages": {
-                "ti": {
-                    "font": {
-                        "family": "GeezWeb",
-                        "source": [
-                            "wookianos.ttf",
-                            "ETHIOPI0.eot",
-                            "wookianos.mobileconfig"
-                        ]
-                    },
-                    "displayName": "Tigrinya",
-                    "languageName": "Tigrinya"
-                }
-            },
-            "lastModifiedDate": "2014-03-21T16:22:19Z",
-            "sourcePath": "legacy/t/tir_et_uni_ez",
-            "encodings": [
-                "unicode"
-            ],
-            "jsFileSize": 13816,
-            "minKeymanVersion": "5.0",
-            "helpLink": "https://help.keyman.com/keyboard/tir_et_uni_ez",
-            "deprecated": true,
```

Comment:
I assume this is intentional, since I'm seeing `deprecated: true` for all keyboards no longer in this fixture?

Reply:
This is intentional -- the search returns only the first 10 results now.